mycli/packages/expanded.py

```diff
@@ -17,7 +28,8 @@ def expanded_table(rows, headers):
     header_len += 2
 
     for row in rows:
-        row_len = max([len(_text_type(x)) for x in row])
+        row = [format_field(x) for x in row]
+        row_len = max([len(x) for x in row])
```

Comment:
Ah. I didn't realize you were mutating the row and assigning it back on to the same var. No need to make it into a generator.